testserver/urls.py

```diff
@@ -4,19 +4,20 @@
 
 from __future__ import absolute_import
 
-from django.conf.urls import url, include
+from django.urls import path, include
 from django.views.i18n import JavaScriptCatalog
-from django.contrib.auth.views import login
+from django.contrib.auth.views import LoginView
 
 from . import views as testserver_views
 
+
 urlpatterns = [  # pylint: disable=invalid-name
-    url(r'^$', testserver_views.index),
-    url(r'^accounts/login/$', login),
-    url(r'^logout/$', testserver_views.logout_page),
-    url(r'^register/$', testserver_views.register),
-    url(r'^register/success/$', testserver_views.register_success),
+    path('', testserver_views.index),
+    path('accounts/login/', LoginView.as_view(template_name='registration/login.html')),
+    path('logout/', testserver_views.logout_page),
+    path('register/', testserver_views.register),
+    path('register/success/', testserver_views.register_success),
 
-    url(r'^api/', include('edx_notifications.server.api.urls')),
-    url(r'^jsi18n/$', JavaScriptCatalog.as_view(), name='javascript-catalog'),
+    path('api/', include('edx_notifications.server.api.urls')),
+    path('jsi18n/', JavaScriptCatalog.as_view(), name='javascript-catalog'),
```

Comment:
I need a little context about removing regex here. we could have used re_path with existing regex.

Reply:
path() always matches the complete path, so path('jsi18n/') is equivalent to url('^jsi18n/$')
So no need to use regexes over here.